src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Validators/Cohort/FileUploadCacheViewModelValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using FluentValidation;
+using SFA.DAS.ProviderCommitments.Web.Models.Cohort;
+
+namespace SFA.DAS.ProviderCommitments.Web.Validators.Cohort
+{
+    public class FileUploadCacheViewModelValidator : AbstractValidator<FileUploadCacheViewModel>
+    {
+        public FileUploadCacheViewModelValidator()
+        {
+            RuleFor(x => x.SelectedOption).NotNull().WithMessage("Select how you want to add apprentices");
```

Comment:
Is this the right error message for this validation?

Reply:
the message isn't correct, fixed it now